### DISCUSSION TITLE
Lora integration

### DISCUSSION THREAD

**Discussion by vorillaz** [microsoft/devicescript#682] [Q&A]
Hi there, really cool project. 
I've started playing around with an ESP32C3 board I had lying around and was wondering if someone has made it to work with Lora . I have a SX1278 module as well and it would be nice to connect two devices with deviscript.

Most projects are using the [arduino-LoRa](https://github.com/sandeepmistry/arduino-LoRa) library so I guess if there is a way to provide a thin wrapper around the project that it could be really nice.

**Comment by pelikhan** [comment]
You would probably want to port or rewrite the lora driver in TypeScript using the SPI services. I don't think we've started on this but we take contributions!
